packages/firestore/.idea/runConfigurations/All_Tests.xml

```diff
@@ -4,9 +4,7 @@
     <node-options />
     <working-directory>$PROJECT_DIR$</working-directory>
     <pass-parent-env>true</pass-parent-env>
-    <envs>
-      <env name="USE_MOCK_PERSISTENCE" value="YES" />
-    </envs>
```

Comment:
I'll send you a PR with the rename and with this change shortly. It didn't surface as an issue since this was only affecting some asserts in SharedClientState.

Reply:
I'll send you a PR with the rename and with this change shortly.